Release-Checklist.md

```diff
@@ -0,0 +1,79 @@
+Change release process, instead create a release candidate we can test the HEAD what will be the last commit using new automated obs package generation we can generate and test commits based on HEAD.
+
+- [ ] Raise PRs to update the `VERSION` file in the following repositories:
+   - [ ] [agent][agent].
+   - [ ] [proxy][proxy].
+   - [ ] [runtime][runtime].
+   - [ ] [shim][shim].
+   - [ ] [throttler][throttler].
+
+  Note that the "phase" element of the project encoded in the version strings needs to match for all components. For example, when a `beta` is released, the version string for *all* components should show `beta`.
+
+- [ ] Ensure all CI tests pass **for all architectures**.
+
+- [ ] Get confirmation from metrics CI that performance is within acceptable limits.
+
+- [ ] Create an **annotated tag** for the new release version for the following repositories:
```

Comment:
A hint/clue/link on how to make/ensure we make an annotated tag here would be nice.

Reply:
And, do we need to define the tag/version format and rules?

Reply:
We can add a link/ or describe that to ensure the process to tag is always the same use 
https://github.com/kata-containers/packaging/pull/16

Reply:
Done (in the main doc to avoid bloating out the checklist).